common/src/main/java/com/huaweicloud/dubbo/common/CommonConfiguration.java

```diff
@@ -46,8 +46,15 @@
   public static final String KEY_REGISTRY_ADDRESS = "dubbo.servicecomb.registry.address";
 
   // ###### config center configuration ############### //
+  public static final String KEY_CONFIG_ADDRESSTYPE = "dubbo.servicecomb.config.addressType";
+
   public static final String KEY_CONFIG_ADDRESS = "dubbo.servicecomb.config.address";
 
+  // ###### kie config center polling configuration############### //
+  public static final String KEY_SERVICE_ENABLELONGPOLLING = "dubbo.servicecomb.enableLongPolling";
```

Comment:
dubbo.servicecomb.config.enableLongPolling & dubbo.servicecomb.config.pollingWaitInSeconds

Reply:
Okay. I'll modify it right now.